server/test/email-templates.spec.es6

```diff
@@ -3,47 +3,154 @@
 require('./common.es6');
 
-describe('Email templates', () => {
-  it('should send an email when noncommercial user cancels', () => {
-    const application = noncommercialPermitApplicationFactory.create();
-    application.status = 'Cancelled';
-    application.type = 'noncommercial';
-    emails.noncommercialApplicationUserCancelled(application);
-    expect(true).to.be.true;
-  });
+// to do convert to ${type}.translateFromClientToDatabase beforeall
+describe('Special use email templates', () =>{
+  const specialUseSubject = 'An update on your recent Open Forest permit application to the Forest Service.';
+  const forestName = 'Mt.Baker - Snoqualmie National Forest';
+  const adminSubject = `A new permit application with a start date of 12/12/2018 has been submitted to the ${forestName}.`;
 
-  it('should send an email when noncommercial user cancels', () => {
-    const application = noncommercialPermitApplicationFactory.create();
-    application.status = 'Review';
-    application.type = 'noncommercial';
-    emails.noncommercialApplicationReview(application);
-    expect(true).to.be.true;
-  });
+  describe('for noncommericial apps', () => {
+    let application = {};
+    beforeEach(function () {
+      // runs before each test in this block
+      let applicationFactory = noncommercialPermitApplicationFactory.create();
+      noncommController.translateFromClientToDatabase(applicationFactory, application);
+      application.forestName = forestName;
+    });
 
-  it('should send an email when temp outfitter application is rejected', () => {
-    const application = tempOutfitterPermitApplicationFactory.create();
-    application.status = 'Cancelled';
-    application.type = 'temp-outfitter';
-    emails.tempOutfitterApplicationRejected(application);
-    expect(true).to.be.true;
-  });
+    it('should build an object of email content for noncommercial app submission to user', () => {
+      const emailContent = emails.noncommercialApplicationSubmittedConfirmation(application);
+      const specialUseSubject = 'Your Noncommercial Open Forest permit application has been submitted for review!';
+      expect(emailContent.subject).to.be.eq(specialUseSubject);
+      expect(emailContent).to.have.all.keys('to','subject', 'body', 'html');
+      
+      expect(emailContent.body.trim()).to.include('NOT APPROVED');
+      expect(emailContent.body.trim()).to.include('Title: Special use administrator');
+      expect(emailContent.body.trim()).to.include('Start date: 12/12/2018 08:00 a.m.');
+      expect(emailContent.body.trim()).to.include('End date: 12/12/2018 04:00 p.m.');
+      expect(emailContent.html.trim()).to.include('<strong>NOT APPROVED</strong>');
+      expect(emailContent.html.trim()).to.include('<td class="border-bottom">Special use administrator</td>');
+      expect(emailContent.html.trim()).to.include('12/12/2018 08:00 a.m.');
+      expect(emailContent.html.trim()).to.include('12/12/2018 04:00 p.m.');
```

Comment:
Part of me wonders about bringing in some kind of HTML snapshotter for testing this instead of looking for contents of the HTML (something like [mocha-snapshots](https://www.npmjs.com/package/mocha-snapshots)).  But honestly I think this case is too simple to justify bringing in some more complex testing.

Reply:
At this point, I'm just trying to make sure that some of the email population is happening (so that when we do refactors we know if we're breaking), so that does fill like adding a bigger lift.